packages/client-sdk-nodejs-compression-zstd/README.md

```diff
@@ -0,0 +1,55 @@
+<img src="https://docs.momentohq.com/img/momento-logo-forest.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+
+
+# Compression Extensions for Momento Node.js SDK
+
+This package provides extensions for the Momento Node.js SDK to support compression and decompression of data that
+is stored using the `CacheClient.set` function and accessed through the `CacheClient.get` function.
+
+To use the library, you will need to install it from npm:
+
+```bash
+npm install @gomomento/sdk-compression
```

Comment:
Is this still the zstd package id?

Reply:
the README.md file gets generated from the README.template.md file. check that one, I think I got everything there.